src/tck/Reactive.Streams.TCK/packages.config

```diff
@@ -1,5 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="NUnit" version="3.2.1" targetFramework="net45" />
-  <package id="Reactive.Streams" version="1.0.1" targetFramework="net45" />
```

Comment:
Shouldn't the TCK depend on the 1.0.0-RC1 version? (Or perhaps that is added automagically?)